types/proposal_test.go

```diff
@@ -21,6 +21,9 @@ var (
 )
 
 func init() {
+	rows, _ := NmtRootsFromBytes([][]byte{[]byte("HeHasBeenElected--June_15_2020_amino_was_removed")})
```

Comment:
How was elected? :-P

Reply:
That was the first random phrase that came into my mind and surprisingly it matched the required length 😆

Reply:
I did not look at the amino)